recipes/schavott/meta.yaml

```diff
@@ -42,3 +41,7 @@ about:
   summary: 'Assembly and scaffolding of bacterial genomes in real time using MinION-sequencing.'
   license_file: LICENSE
 
+extra:
+  container:
+    # matplotlib depends on libGL.so.1 from the system
```

Comment:
@druvus this is only true for some plotting backends. I think with the Agg backend everything should work.